jstest/testrunner/devices/rpi2.py

```diff
@@ -19,4 +19,4 @@ class RPi2Device(SSHDevice):
     Device of the Raspberry Pi 2 target.
     '''
     def __init__(self, env):
-        SSHDevice.__init__(self, env, 'linux')
+        SSHDevice.__init__(self, env, 'linux', 'pi@raspberrypi:~$')
```

Comment:
The prompt must be a constant string. Serial devices have constant prompts: `nsh>` and `TASH>`. SSH devices have more complex OS where the prompt is not a constant. For example, if you use Raspbian OS with a `test` user, the prompt is `test@raspberry:~$`.